test/parser-test.js

```diff
@@ -26,9 +26,13 @@ describe('Parser', () => {
     it('can parse a basic ruleset', () => {
```

Comment:
Stylistic note: consider because nodeFactory is used *so much* in this file, I think it may be a net win for readability if you shortened it to something like nF, just as a reduction of the visual noise and required structural whitespace.